network/definition/data/templates/signals.yaml

```diff
@@ -95,3 +95,21 @@ signals:
         min: 0.0
         max: 15
     continuous: true
+  wheelLinearVelocity:
+    unit: 'm/s'
+    nativeRepresentation:
+      bitWidth: 10
+      resolution: 0.01
+      range:
+        min: 0
+        max: 50
```

Comment:
Do you need to specify all three of `bitWidth`, `resolution`, and `range.max`? I think yamcan can auto-calculate the most optimal settings if you omit some of these. Namely, in this case, probably fine to use just `range` and `resolution`